apps/staking/components/StakedNodeCard.tsx

```diff
@@ -169,10 +179,12 @@ const ReadyForExitNotification = ({
   date,
   timeString,
   className,
+  isDeregistered,
```

Comment:
Small thing but I usually have className as the last prop?

Reply:
Not a big deal. It seems to vary in the code base so you can resolve this comment without changes if you want.